### DISCUSSION TITLE
HA dashboard stops accepting touch input from Fully Kiosk. Not sure where to start.

### DISCUSSION THREAD

**Discussion by fcassata** [seaky/nspanel_pro_tools_apk#112] [Q&A]
I have two panels, both running fully kiosk and two different and pretty basic one screen HA dashboards. After a few minutes, one of them (FW 2.2) stops sending touch input through the webview, but I can still do the 5 x tap to enter the fully kiosk pin, so the panel itself is registering touch. Everything is fine if I just run the HA companion app, but I'd much rather keep using fully for the remote management. The A-OK one is FW 1.7.

Fully settings are identical, I did a diff on the export. The display is not sleeping or dimmed when this happens, or coming out of sleep.

Any ideas how to even start debugging this?

**Comment by seaky** [comment]
Hi,
What does it mean “stops sending touch input through the webview”? “Sending” to where?
Touch gestures only works on black screen and the device fw must be below 1.11.

**Reply by fcassata** [reply]
Ahh sorry for being unclear, I'm just talking about touch in general. At first I though the panel was frozen since the display was on, the dashboard was there, but I couldn't interact with it, couldn't tap buttons or slide sliders. But the Fully Kiosk unlock I have, 5 x tap, DID work, so that confirmed it probably has something to do with the webview and/or the actual dashboard.

And the issue actually moved over to the other panel when loading the offending dashboard there, so it looks like it's not a matter of an issue with the different FW versions. But it's interesting that it only happens in Fully Kiosk and not in HA Companion. Over on HA forum someone suggested it had to do with the webview version installed.

Is this totally off-topic to discuss here? If so I'll take it to the HA forums instead so I'm not cluttering this discussion section with irrelevant problems :)
